splunk-otel-android/src/main/java/com/splunk/rum/RumAttributeAppender.java

```diff
@@ -84,9 +89,18 @@ public void onStart(Context parentContext, ReadWriteSpan span) {
         span.setAttribute(SplunkRum.SCREEN_NAME_KEY, currentScreen);
         CurrentNetwork currentNetwork = connectionUtil.getActiveNetwork();
         span.setAttribute(NET_HOST_CONNECTION_TYPE, currentNetwork.getState().getHumanName());
+        appendNetworkAttributes(span, currentNetwork);
+    }
+
+    @SuppressWarnings("NullAway")
+    static void appendNetworkAttributes(Span span, CurrentNetwork currentNetwork) {
```

Comment:
Should we include `NET_HOST_CONNECTION_TYPE` too? It's being set in both places

Reply:
Great catch! Will do.